README.md

```diff
@@ -0,0 +1,106 @@
+# 🌱  Fresh environment
+
+A **Fresh** environment is a fast and ready-to-use Docker container with
+various developer tools pre-installed. Including Node.js, and headless
+browsers. It aims to help to run npm packages on your machine,
+_without_ putting your personal data at risk!
+
+## Quick start
+
+Run the following from a terminal:
+
+```sh
+curl -s https://raw.githubusercontent.com/wikimedia/fresh/19.09.0/bin/fresh-node10 \
+> /usr/local/bin/fresh-node \
+&& curl -s https://raw.githubusercontent.com/wikimedia/fresh/19.09.0/bin/networked-fresh-node10 \
+> /usr/local/bin/networked-fresh-node \
+&& chmod +x /usr/local/bin/fresh-node /usr/local/bin/networked-fresh-node \
+&& echo -e '\n\xf0\x9f\x8c\xb1\x20Fresh\x2019.09.0\x20is\x20now\x20ready\x21\n'
+```
+
+This will save [fresh-node](/bin/fresh-node10) and [networked-fresh-node](/bin/networked-fresh-node10) to `/usr/local/bin/`, and marks these files as executable.
+
+Programs in this directory automatically become commands you can use from your terminal.
+
+### What's inside
+
+* **Node.js** 10 with npm 6
+* **Firefox**
+* **Chromium**
+* `chromedriver`, `ffmpeg`, and `xvfb` (for browser tests)
+* JSDuck
+* Debian 9 Jessie (kernel)
+
+### Prerequisites
+
+You'll need to have Docker installed. See [Docker CE for Linux](https://docs.docker.com/install/#server), [Docker for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac), or [Docker for Windows](https://docs.docker.com/docker-for-windows/install/).
```

Comment:
I might note here that for networked-fresh-node-10, only linux support exists per https://docs.docker.com/network/network-tutorial-host/

"The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, Docker Desktop for Windows, or Docker EE for Windows Server."

Reply:
Interesting. I'm using it on Mac though, regularly. `networked-fresh-node10` is able to access my other localhost services, while `fresh-node` is not.

Reply:
Huh, must be old doc on Docker's part.